cypress/e2e/demo/errors.cy.ts

```diff
@@ -1,10 +1,12 @@
 context('Errors', () => {
   function checkErrorReported(type: string, value: string, expectStacktrace = true) {
-    cy.waitExceptions((events) => {
-      expect(events).to.have.lengthOf(1);
-      const exception = events[0];
+    cy.waitExceptions((evts) => {
```

Comment:
did the word `event` piss off a linter? :D

Reply:
Neah. Mass-replaced "event" with "evt", including for the plurals. It's a good practice to avoid using `event` as it is also a global variable hence it can cause errors (i.e. removing the parameter from a function and the build would just work).